sn_networking/src/cmd.rs

```diff
@@ -546,9 +558,6 @@ impl SwarmDriver {
 
                 let _ = sender.send(closest_peers);
             }
-            // SwarmCmd::GetOurCloseGroup { sender } => {
-            //     let _ = sender.send(self.close_group.clone());
-            // }
```

Comment:
why keep it as a comment?

Reply:
This removes the comment that was left some other time.

Reply:
Oh right!! 💯